licenses.d2iq.yaml

```diff
@@ -46,6 +46,7 @@ ignore:
   - docker.io/bitnami/kubectl:1.24.1
   - docker.io/bitnami/kubectl:1.26.4
   - docker.io/bitnami/kubectl:1.27.9
+  - docker.io/bitnami/kubectl:1.29.2
```

Comment:
for my own edification, do you know why we ignore these ?

Reply:
```  # Incorrect mapping was provided
  # - container_image: docker.io/bitnami/kubectl:1.19.7
  #   sources:
  #     - url: https://github.com/bitnami/bitnami-docker-kubectl
  #       ref: ${image_tag}-debian-10-r31
  #       license_path: LICENSE```

might explain it in this block? (but I'm not really sure either)